File: solutions/beecrowd/1011/1011.dart
import 'dart:io';
import 'dart:math';

void main() {
  double r, pi = 3.14159;
  var line;

  while ((line = stdin.readLineSync()) != null) {
    r = double.parse(line);
    print('VOLUME = ${(4.0 / 3.0 * pi * pow(r, 3)).toStringAsFixed(3)}');
  }
}

File: solutions/beecrowd/1011/1011.clj
(ns main)

(def pi 3.14159)

(defn main []
  (loop [line (read-line)]
    (when line
       (let [radius (Double/parseDouble line)]
         (printf "VOLUME = %.3f%n"
                 (-> (/ 4.0 3.0)
                     (* pi (Math/pow radius 3)))))
       (recur (read-line)))))

(main)
